docker.gradle

```diff
@@ -22,14 +23,21 @@ ext {
     dockerTimeout = project.hasProperty('dockerTimeout') ? dockerTimeout.toInteger() : 840
     dockerRetries = project.hasProperty('dockerRetries') ? dockerRetries.toInteger() : 3
     dockerBinary = project.hasProperty('dockerBinary') ? [dockerBinary] : ['docker']
+    dockerBuildArgs = project.hasProperty('dockerBuildArgs') ? dockerBuildArgs : ''
 }
 
 if(project.hasProperty('dockerHost')) {
     dockerBinary += ['--host', project.dockerHost]
 }
 
 task distDocker << {
-    def cmd = dockerBinary + ['build', '-t', dockerImageName, project.buildscript.sourceFile.getParentFile().getAbsolutePath()]
+    def cmd
+    if (dockerBuildArgs != '') {
+        cmd = dockerBinary + ['build', '--build-arg', dockerBuildArgs, '-t', dockerImageName, project.buildscript.sourceFile.getParentFile().getAbsolutePath()]
+    }
+    else {
+        cmd = dockerBinary + ['build', '-t', dockerImageName, project.buildscript.sourceFile.getParentFile().getAbsolutePath()]
+    }
```

Comment:
@markusthoemmes - Oh.  I had been thinking that ansible was called prior to the gradle build.  If that's not the case, this logic is moot.  It looks like the build timestamp is set in the whisk.properties during the gradle build.  I'll use that instead - so, no need for any ansible updates re: setting this property in a default.props file (like the python CLI does)